examples/request_redraw_threaded.rs

```diff
@@ -1,6 +1,9 @@
+#[cfg(not(target_arch = "wasm32"))]
 use std::{thread, time};
 
+#[cfg(not(target_arch = "wasm32"))]
 use simple_logger::SimpleLogger;
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
This doesn't look all that great, but it's not that big of an issue either.

Reply:
> I agree but what's the alternative ?

Realistically, there's the option of moving the imports into the `main` function. There's also the option of using the `xtask` pattern to work around this, but that's more work.

Reply:
I moved the imports in the `main` function.